app/models/option.rb

```diff
@@ -0,0 +1,4 @@
+class Option < ApplicationRecord
```

Comment:
Option is a very generic name for this. Now that we have changed what used to be `Outcomes` to `UserBet` I think outcome more accurately describes what this model represents